lxd/db/certificates.mapper.go

```diff
@@ -67,23 +67,11 @@ func (c *ClusterTx) GetCertificates(filter CertificateFilter) ([]Certificate, er
 	// Result slice.
 	objects := make([]Certificate, 0)
 
-	// Check which filter criteria are active.
-	criteria := map[string]interface{}{}
-	if filter.Fingerprint != "" {
-		criteria["Fingerprint"] = filter.Fingerprint
-	}
-	if filter.Name != "" {
-		criteria["Name"] = filter.Name
-	}
-	if filter.Type != -1 {
-		criteria["Type"] = filter.Type
-	}
-
 	// Pick the prepared statement and arguments to use based on active criteria.
 	var stmt *sql.Stmt
 	var args []interface{}
 
-	if criteria["Fingerprint"] != nil {
+	if filter.Fingerprint != nil {
```

Comment:
@masnax as mentioned on IRC, what is the plan here with this function (and others) ignoring certain filter fields (like `Type`) here?